google-cloud-spanner/Gemfile

```diff
@@ -11,7 +11,7 @@ gem "google-cloud-spanner-admin-instance-v1", "~> 1.5"
 gem "google-cloud-spanner-v1", "~> 1.5"
 gem "google-style", "~> 1.31.0"
 gem "grpc", "~> 1.67"
-gem "grpc-tools", "~> 1.67"
+gem "grpc-tools", "<= 1.73"
```

Comment:
grpc has been fixed as of 1.74.1. Please use:

```ruby
gem "grpc-tools", "~> 1.74", ">= 1.74.1"
```

This means "any 1.x where x >= 74, AND later or equal to 1.74.1"